[FEATURE REQUEST] `Properties` derive macro shouldn't import `PropertyGet`

`Properties` [imports](https://gtk-rs.org/gtk-rs-core/stable/0.20/docs/src/glib_macros/properties.rs.html#761) [`PropertyGet`](https://gtk-rs.org/gtk-rs-core/stable/0.20/docs/glib/property/trait.PropertyGet.html) into an outer scope, changing the method resolution behavior.

Recently I learned about the [`Properties` derive macro](https://gtk-rs.org/gtk-rs-core/stable/0.20/docs/glib/derive.Properties.html), and when I tried to replace manual implementations with that, I got type mismatch errors in nearby code:

```
error[E0277]: expected a `Fn(&std::vec::Vec<std::string::String>)` closure, found `usize`
   --> xxxxxxxx.rs:99:51
    |
 99 |                                           foo.get(i)
    |                                               --- ^ expected an `Fn(&std::vec::Vec<std::string::String>)` closure, found `usize`
    |                                               |
    |                                               required by a bound introduced by this call
    |
    = help: the trait `for<'a> Fn(&'a std::vec::Vec<std::string::String>)` is not implemented for `usize`
note: required by a bound in `glib::property::PropertyGet::get`
   --> xxx/.cargo/registry/src/index.crates.io-6f17d22bba15001f/glib-0.20.2/src/property.rs:67:18
    |
67  |     fn get<R, F: Fn(&Self::Value) -> R>(&self, f: F) -> R;
    |                  ^^^^^^^^^^^^^^^^^^^^^ required by this bound in `PropertyGet::get`
```

Before the change, the `get` call was resolved to `<[String]>::get` through a deref of `foo: &Vec<String>`. With `PropertyGet` in scope, it gets resolved to [`<Vec<String> as PropertyGet>::get`](https://gtk-rs.org/gtk-rs-core/stable/0.20/docs/glib/property/trait.PropertyGet.html#tymethod.get).

--- a/glib-macros/src/properties.rs
+++ b/glib-macros/src/properties.rs
@@ -758,8 +758,6 @@ pub fn impl_derive_props(input: PropsMacroInput) -> TokenStream {
     };
 
     let expanded = quote! {
-        use #crate_ident::property::{PropertyGet, PropertySet};
-
         #properties_enum
 
         impl #crate_ident::subclass::object::DerivedObjectProperties for #struct_ident {

--- a/glib-macros/tests/properties.rs
+++ b/glib-macros/tests/properties.rs
@@ -182,6 +182,19 @@ mod foo {
                     .expect("Setter to be only called once");
             }
         }
+
+        /// Checks that `Properties` does not pollute the scope with
+        /// trait imports, as it did in older versions.
+        #[test]
+        fn no_import_leaks() {
+            // `vec.get` can match these methods (in order of precedence):
+            // (1) `<Vec<String> as PropertyGet>::get`
+            // (2) `<[String]>::get` through deref of `Vec<String>`
+            // Had the macro introduced `PropertyGet` into the scope, it would
+            // resolve to (1), which we do not want.
+            let vec: Vec<String> = vec![String::new(); 2];
+            assert_eq!(vec.get(1), Some(&String::new()));
+        }
     }
 
     glib::wrapper! {

